source/linux/Foundational_Components/U-Boot/UG-HSM.rst

```diff
@@ -0,0 +1,123 @@
+###############
+HSM M4F Loading
+###############
+
+   Some K3 SoCs like J721S2 and J784S4 have a High Security Module (HSM) M4F
```

Comment:
Please do not indent section contents. While it is allowed, and doesn't really break anything, it adds unnecessary indentation to the section.